docs/PCF8574--IOExt0_IOExt1_IOExt2-infos.md

```diff
@@ -28,3 +28,8 @@ Usage in car hull:
 - GND, VCC, I2C (SDA,SCL), Interrupt
 
 ![img](.PCF8574--IOExt0_IOExt1_IOExt2-infos/IO-Ext-CurcuitDiagram.drawio.png)
+
+### Board Assembly
+
+![image-20210823131457974](.PCF8574--IOExt0_IOExt1_IOExt2-infos/image-20210823131457974.png)
+
```

Comment:
Please rename and add them to a non-hidden folder.

Reply:
done